.circleci/config.yml

```diff
@@ -3,12 +3,38 @@ version: 2.1
 orbs:
   hmpps: ministryofjustice/hmpps@3.0
   owasp: entur/owasp@0.0.10
+executors:
+  validator-integration:
+    docker:
+      - image: circleci/openjdk:11-jdk
+      - image: localstack/localstack:0.11.2
+        environment:
```

Comment:
I've never checked if we actually need all this, or is it just magic cruft that we copy around.

Reply:
I will try taking bits out on a separate branch and see what happens